src/main/resources/camunda/camunda-historic-task-unterminated-query.json

```diff
@@ -0,0 +1,18 @@
+{
+  "taskDefinitionKey": "processTask",
+  "processDefinitionKey": "wa-task-initiation-ia-asylum",
```

Comment:
This works for now for IA only.

As soon as we work with other jurisdictions then this will have to be refactored as it won't query process.  

We should probably have this configured in it's own file and inserted here for R2 Jobs so we can loop through all of the onboarded jurisdictions and caseTypes.

We should put a ticket in the backlog for that since I don't think we called it out for this ticket.

Reply:
I'll inform Craig regarding this point. We'll raise a ticket for this